package.json

```diff
@@ -18,22 +18,36 @@
   "license": "ISC",
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.13.0",
-    "@angular/cli": "7.3.10",
-    "@angular/common": "7.2.16",
-    "@angular/compiler": "7.2.16",
-    "@angular/compiler-cli": "7.2.16",
-    "@angular/core": "7.2.16",
-    "@angular/forms": "7.2.16",
-    "@angular/http": "7.2.16",
-    "@angular/platform-browser": "7.2.16",
-    "@angular/platform-browser-dynamic": "7.2.16",
-    "ng-packagr": "^5.7.1",
-    "rxjs": "6.6.7",
-    "rxjs-tslint": "^0.1.8",
-    "typescript": "3.2.4",
-    "zone.js": "~0.8.26"
+    "@angular/cli": "7.x",
+    "@angular/common": "7.x",
+    "@angular/compiler": "7.x",
+    "@angular/compiler-cli": "7.x",
+    "@angular/core": "7.x",
+    "@angular/forms": "7.x",
+    "@angular/platform-browser": "7.x",
+    "@angular/platform-browser-dynamic": "7.x",
+    "ng-packagr": "5.x",
+    "rxjs": "6.x",
+    "rxjs-tslint": "0.1.x",
+    "typescript": "3.x",
+    "zone.js": "0.8.x"
+  },
+  "peerDependencies": {
+    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular/cli": "7.x",
+    "@angular/common": "7.x",
+    "@angular/compiler": "7.x",
+    "@angular/compiler-cli": "7.x",
+    "@angular/core": "7.x",
+    "@angular/forms": "7.x",
+    "@angular/platform-browser": "7.x",
+    "@angular/platform-browser-dynamic": "7.x",
+    "ng-packagr": "5.x",
+    "rxjs": "6.x",
+    "rxjs-tslint": "0.1.x",
+    "typescript": "3.x",
+    "zone.js": "0.8.x"
```

Comment:
a good number of these dependencies are only required to _build_/_compile_ the project, and aren't needed to run the project.

Specifically, `@angular-devkit/build-angular`, `@angular/compiler`, `@angular/compiler-cli`, `ng-pacakgr`, `rxjs-tslint`, are all dependencies I _know_ are only required for the build, and consumers shouldn't need to install them.

I'm unsure about a few of the other packages, for different reasons.

This project previously only had `@angular/common`, `@angular/core`, and `@angular/forms` as peer dependencies. I'm pretty confident that those are the only dependencies consumers will _need_ to have installed.

Reply:
Updated according to the above recommendation.